controls/compute.sp

```diff
@@ -0,0 +1,293 @@
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      disk.id as resource,
+      case
+        when disk_iops_read_write > 16000 then 'alarm'
+        else 'ok'
+      end as status,
+      disk.title || ' has ' || disk_iops_read_write || ' IOPS.'
+      as reason,
+      disk.resource_group,
+      sub.display_name as subscription
+    from
+      azure_compute_disk as disk,
+      azure_subscription as sub
+    where
+      sub.subscription_id = disk.subscription_id;
+  EOT
+
+  tags = merge(local.compute_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "compute_disk_large" {
+  title       = "Disks with over 100 GB should be resized if too large"
+  description = "Large compute disks are unusual, expensive and should be reviewed."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      disk.unique_id as resource,
+      case
+        when disk_size_gb <= 100 then 'ok'
+        else 'alarm'
+      end as status,
+      disk.title || ' is ' || disk_size_gb || ' GB.' as reason,
+      resource_group,
+      sub.display_name as subscription
+    from
+      azure_compute_disk as disk,
+      azure_subscription as sub
+    where
+      sub.subscription_id = disk.subscription_id;
+  EOT
+
+  tags = merge(local.compute_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "compute_disk_low_iops" {
+  title       = "Disks with low IOPS should be reviewed"
+  description = "Compute disks with low IOPS should be reviewed."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      disk.id as resource,
+      case
+        when disk_iops_read_write <= 1500 then 'alarm'
+        else 'ok'
+      end as status,
+      disk.title || ' has ' || disk_iops_read_write || ' IOPS.' as reason,
+      disk.resource_group,
+      sub.display_name as subscription
+    from
+      azure_compute_disk as disk,
+      azure_subscription as sub
+    where
+      sub.subscription_id = disk.subscription_id;
+  EOT
+
+  tags = merge(local.compute_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "compute_disk_snapshot_storage_standard" {
+  title       = "Disks snapshots storage type should be standard"
+  description = "Use standard storage type for compute disks snapshots to save cost."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      ss.id as resource,
+      case
+        when ss.sku_tier = 'Standard' then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when ss.sku_tier = 'Standard' then ss.title || ' has storage type ' || ss.sku_tier || '.'
+        else ss.title || ' has storage type ' || ss.sku_tier || '.'
+      end as reason,
+      ss.resource_group,
+      sub.display_name as subscription
+    from
+      azure_compute_snapshot as ss,
+      azure_subscription as sub
+    where
+      ss.subscription_id = sub.subscription_id;
+  EOT
+
+  tags = merge(local.compute_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "compute_disk_standard_hdd" {
```

Comment:
@khushboo9024 Is this a general recommendation for cost saving? What if a user actually needs an SSD disk for their application? In the AWS volume type cases, going from GP2 -> GP3 was a definitive cost saver since GP3 was the next generation of GP2 volumes.